workflow/log-monitoring/search-filter.mdx

```diff
@@ -0,0 +1,38 @@
+---
+title: Search & Filter Your Logs
+sidebarTitle: Search & Filter
+---
+
+The Search and Filter features within the Logs page provide a streamlined method for accessing your data. Create persistent filters to preserve your customized views, efficiently browse through your logs, and access important collections of logs in a single button click.
+
+To quickly filter your log data, use the leftmost column on the logs page to toggle on and off filters sorted by __Levels__, __Additional__, __NameSpace__, __APM Projects__, and __Sources__.
+
+# Create New Filters
```

Comment:
Would recommend changing this to "Save Filter Combinations", because technically we're calling the left list all filters, and nobody can add to that list manually.